src/pgrepr/src/value.rs

```diff
@@ -399,7 +399,12 @@ impl Value {
                 buf.put_u32(elem_type.oid());
                 for dim in dims {
                     buf.put_i32(pg_len("array dimension length", dim.length)?);
-                    buf.put_i32(pg_len("array dimension lower bound", dim.lower_bound)?);
+                    buf.put_i32(dim.lower_bound.try_into().map_err(|_| {
+                        io::Error::new(
```

Comment:
Uncovered

Reply:
I don't see these conditions covered in any tests; this is a fairly exotic error case so I am not too apprehensive though maybe you have different feelings?

Reply:
Fine by me then.